src/viking.js

```diff
@@ -1,11 +1,74 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    this.attack = function (){
+        return this.strength;
+    };
+    this.receiveDamage = function (damage) {
+        this.health -= damage; 
+    }
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+    Soldier.call(this,health, strength);
+    this.name = name;
+    this.receiveDamage =function (damage) {
+        this.health -= damage;
+        if(this.health > 0) {
+            return `${this.name} has received ${damage} points of damage`
+        } else {
+            return `${this.name} has died in act of combat`
+        }
+    }
+    this.battleCry = function () {
+        return "Odin Owns You All!"
+    }
+}
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
```

Comment:
Here you link the Viking prototype to the Soldier prototype but as you didn't create the methods on the Soldier prototype, it's actually empty (if you try console logging Soldier.prototype), so it's not really needed ;)